### DISCUSSION TITLE
Existence of Galaxies. Newton vs Einstein

### DISCUSSION THREAD

**Discussion by khuramonline** [a-cosmology-group/acg#131] [Astrophysics and Physics]
Einstein had basically failed to reach to the idea of galaxies. However, Newton had duly reached to the idea about galaxies. I have explained in this [blog post](https://bbtrejected.wordpress.com/2019/11/07/gr-failed-in-predicting-the-existence-of-galaxies-newton-however-reached-to-this-kind-of-structure-i-e-galaxy/).

**Comment by Francois-Zinserling** [comment]
Until around 1920 we still thought the Milky Way **was** the entire universe. It consisted of stars, stars, and more stars, and then some Messier's fuzzballs. 
It was not that long ago ...

Even though telescopes were not new around then, Doppler measurements of starlight was a new field, and distance scales beyond local stars were a fairly new science. Andromeda was still called a nebula.

So much so that when 'distant galaxies' were confirmed to exist, (after 1920) they were at first referred to as 'separate universes'
And we only really confirmed ourselves as being within our own galaxy when we were able to properly identify other fuzzballs as galaxies.
It took Hubble to kickstart this lot around 1924.

So not to sound as if I am in defense of Einstein, but the term 'galaxy' and 'universe' meant more or less the same thing when he did GR in 2016.

And to play devil's advocate; I am neither here nor there on the universe being infinite, but I generally have a problem with infinites in physics; 
You don't need infinity to contain a 'universe'. If a spiral galaxy can be contained within bounds, then so can a 'universe'.

**Reply by HanDeBruijn** [reply]
>I am neither here nor there on the universe being infinite, but I generally have a problem with infinites in physics

Let us assume that the universe ***is*** "infinite" in space and time. Because of [Russel's Paradox](https://en.wikipedia.org/wiki/Russell%27s_paradox) ( $A \in A \Rightarrow A \notin A$ ) and [Infinitum Actu Non Datur](http://www.alternatievewiskunde.nl/MijnBoek/oneindig/index.htm) this effectively means that **[There is no universe](https://medium.com/@petergleeson1/there-is-no-universe-cec835dc575f)  at all**, in the sense that there would exist something that contains everything. Our _infinite universe_, according to [Gauss](https://en.wikipedia.org/wiki/Carl_Friedrich_Gauss)' dictum, is just _a way of speaking_.

**Comment by OlliSant** [comment]
About infinity: there is the longest possible distance in the universe. That is a better definition of infinity as infinity.

**Reply by RedshiftDrift** [reply]
That's not the definition of 'infinity'.  Redefining a word only causes greater confusion.

**Reply by OlliSant** [reply]
The question is: What kind of space is the space of the universe? Not: What infinity means? The space of the universe is infinite, because it is so immensely big, that there is no real difference between a big size and infinity, but finite because there is all the space that exists, and there is no outside place in any sense, and no edge, and these can be understood as infinite things too as well as finite things.

Infinity means mathematical infinity, logical infinity, absolute infinity, but the infinity of the universe is a rel thing in the real life, and is a different kind of infinity.

There is no sound cosmology if we don't understand the infinity and eternity of the universe as such as they are in the reality of the universe. Or if we think that universe is not infinite and eternal, because of definitions in mathematics.

**Reply by HanDeBruijn** [reply]
@OlliSant . Almost, but not quite. Crude definition: _a thing is infinite if there is always something **beyond** it_.
Adopting this **potential infinity** instead of (alas) the standard [Actual infinity](https://en.wikipedia.org/wiki/Actual_infinity#Aristotle's_potential%E2%80%93actual_distinction) makes a lot of "subtle" mathematics obsolete. No problem. Such math has never been relevant for science (e.g. physics) anyway.

**Reply by OlliSant** [reply]
We make theories of the space of the universe, and then we test them. Theory of the space, mathematics of the space, then we test the theory. My idea of the mathematics here is simple: 4D, ball- like space, boundless, and its diameter, Q. Q is the longest possible distance in the universe. Unknown distance, but we can make guesses of it and test them. 4D and Q and the matter is galaxies and forces and radiations and explosions. No beginning, no end.

4D, Q, Galaxies, forces, explosions, 

Just change the GR equations to this model. Then test it and make better.
